README.md

```diff
@@ -73,10 +73,10 @@ You can use `--task`/`-t` flag to create a new task.
 $ ct --task MyProject Updated README with the new details
 ```
 
-If the project that you about to create is not exists then you have to specify the flag `--create`/`--cp` for CommitTasks to understand that you want to create a new project and not that you're having a typo in your project name.
+If the project that you are about to create does not exist then you must specify the flag `--create`/`--cp` for CommitTasks to understand that you want to create a new project.
 
 ```
-$ ct --task ProjectThatNotExisted Improve documentation --create
+$ ct --task ProjectToBeCreateded Improve documentation --create
```

Comment:
First of all, thank you for this PR, it must be annoying for you to read this readme :smile: . I believe the word here is `ProjectToBeCreated`

Reply:
Fixed. Not annoying at all. It was perfectly understandable. I like the looks of your project, and this was just an easy way I could help. Good luck with it.